test/test_clinic_finder.rb

```diff
@@ -10,4 +10,24 @@ def setup
   def test_that_initialize_sets_clinic_variable
   	assert_kind_of Hash, @abortron.clinics
   end
+
```

Comment:
this looks sooooo good. If there's anything else that needs a test can you leave a comment, just so we have a record of where to add tests after the fact?